### PR TITLE
display review date in ReviewWork and ReviewApplication Panels

### PR DESCRIPTION
**Pull request by rkzel**
Original request slightly changed:

Screenshot from 2019-11-22 20-19-03

This is the almost correct version, the final version has lighter grey color of the dates. Also, the change was made to both review panels - Work and Application.

### CODE REVIEW

File: apps/projects/app/components/Panel/ReviewApplication/ReviewApplication.js [hunk 1]
```diff
@@ -135,17 +135,22 @@ const ReviewApplication = ({ issue, requestIndex }) => {
...
+              ) : (
+                <>
+                  <IconCross color={`${theme.negative}`} css="margin-top: -4px; margin-right: 8px" />
+                  <Text color={`${theme.negative}`}>Rejected</Text>
+                </>
+              )}
+              <Text color={`${theme.contentSecondary}`} css={`margin-left: ${GU}px`}>
+                {formatDate(new Date(request.review.reviewDate), 'd MMM yy, h:MM a (z)')}
+              </Text>
+            </div>
```
Comment (chadoh) [inline apps/projects/app/components/Panel/ReviewApplication/ReviewApplication.js]: I think we could clean this up a bit by creating a new `Status` component.

```js
const statuses = [
  { color: 'negative', text: 'Rejected' },
  { color: 'positive', text: 'Accepted' },
]

const Status = ({ review }) => {
  const theme = useTheme()
  const status = statuses[Number(review.approved)]
  const text = status.text
  const color = theme[status.color]
  return (
    <div css="display: flex; align-items: center">
      <IconCheck color={color} css={`margin-top: -${0.5 * GU}px; margin-right: ${GU}px`} />
      <Text color={color}>{text}</Text>
      <Text color={`${theme.contentSecondary}`} css={`margin-left: ${GU}px`}>
        {formatDate(new Date(review.reviewDate), 'd MMM yy, h:MM a (z)')}
      </Text>
    </div>
  )
}
```

File: apps/projects/app/components/Panel/ReviewWork/ReviewWork.js [hunk 2]
```diff
@@ -124,17 +123,22 @@ const ReviewWork = ({ issue }) => {
...
+              ) : (
+                <>
+                  <IconCross color={`${theme.negative}`} css="margin-top: -4px; margin-right: 8px" />
+                  <Text color={`${theme.negative}`}>Rejected</Text>
+                </>
+              )}
+              <Text color={`${theme.contentSecondary}`} css={`margin-left: ${GU}px`}>
+                {formatDate(new Date(work.review.reviewDate), 'd MMM yy, h:MM a (z)')}
+              </Text>
+            </div>
```
Comment (chadoh) [inline apps/projects/app/components/Panel/ReviewWork/ReviewWork.js]: We could put this new `Status` component in `components/Panel` or maybe in `components/Shared` and then use it in both of these spots.

File: apps/projects/app/components/Panel/ReviewApplication/ReviewApplication.js (hunk 1 above)
Comment (rkzel) [reply apps/projects/app/components/Panel/ReviewApplication/ReviewApplication.js]: Done in PR #1698

File: apps/projects/app/components/Panel/ReviewWork/ReviewWork.js (hunk 2 above)
Comment (rkzel) [reply apps/projects/app/components/Panel/ReviewWork/ReviewWork.js]: Done in PR #1698

**Comment by stellarmagnet** [comment]
@rkzel @chadoh What's left on this?

File: apps/projects/app/components/Panel/ReviewApplication/ReviewApplication.js (hunk 1 above)
Comment (chadoh) [reply apps/projects/app/components/Panel/ReviewApplication/ReviewApplication.js]: Do these two PRs have conflicts, then? I'm not sure I understand the intended workflow.

**Pull request merged**